dev-libs/serdisplib/serdisplib-2.01.ebuild

```diff
@@ -0,0 +1,102 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools udev
+
+DESCRIPTION="Library to drive several displays with built-in controllers or display modules"
+HOMEPAGE="http://serdisplib.sourceforge.net"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="deprecated dlo experimental sdl threads tools usb"
+REQUIRED_USE="lcd_devices_acoolsdcm? ( usb )
+		lcd_devices_directgfx? ( sdl )
+		lcd_devices_glcd2usb? ( usb )
+		lcd_devices_displaylink? ( deprecated dlo )
+		lcd_devices_remote? ( experimental )"
+
+# Define the list of valid lcd devices.
+IUSE_LCD_DEVICES=( acoolsdcm ddusbt directgfx displaylink framebuffer glcd2usb
+			goldelox i2c ks0108 l4m lc7981 lh155 nokcol pcd8544
+			remote rs232 sed133x sed153x sed156x ssdoled stv8105 t6963 )
+
+# Add supported drivers from 'IUSE_LCD_DEVICES' to 'IUSE' and 'REQUIRED_USE'
+REQUIRED_USE="${REQUIRED_USE} || ("
+for lcd_driver in "${IUSE_LCD_DEVICES[@]}"; do
```

Comment:
You can save a lot of lines and that loop using $(printf 'lcd_devices_%s ' ${IUSE_LCD_DEVICES[@]})

Reply:
Thanks, sounds very good, updated in newest push.